openrtb_ext/request.go

```diff
@@ -149,6 +150,28 @@ type ExtRequestPrebidCacheVAST struct {
 	ReturnCreative *bool `json:"returnCreative,omitempty"`
 }
 
+type ExtRequestPrebidBidAdjustments struct {
+	MediaType MediaType `json:"mediatype,omitempty"`
+}
+
+// How the map is strucutred
+// Map #1 -- Key: BidderName, Value: Another Map
+// Map #2 -- Key: DealID, Value: Adjustments Array
+// Overall: BidderName maps to a DealID that maps to an Adjustments Array
+type MediaType struct {
+	Banner   map[string]map[string][]Adjustments `json:"banner,omitempty"`
+	Video    map[string]map[string][]Adjustments `json:"video,omitempty"`
+	Audio    map[string]map[string][]Adjustments `json:"audio,omitempty"`
+	Native   map[string]map[string][]Adjustments `json:"native,omitempty"`
+	WildCard map[string]map[string][]Adjustments `json:"*,omitempty"`
+}
+
+type Adjustments struct {
+	AdjType  string  `json:"adjtype,omitempty"`
```

Comment:
Consider calling this field name `Type` since this is already within an adjustment object.